tutorials/Application.setShortcuts.md

```diff
@@ -0,0 +1,26 @@
+Sets new application's shortcut configuration
+
+# Example
+```js
+async function createWin(config) {
+		const app = await fin.Application.create({
+			uuid: 'app-1',
+			name: 'myApp',
```

Comment:
Shouldn't `uuid` & `name` be the same?

Reply:
I would not matter because it does not really because the application `uuid` is given